extension/ts/main/extensions/author/authorPage/authorUI.ts

```diff
@@ -50,44 +50,52 @@ const createEditTagsSection = ({onSave, onPull, onCancel}: ButtonHandlers) => {
 	return section;
 };
 
-const createCurrentTagsButtons = ({onPush, onSync, onEdit}: ButtonHandlers, getTagsCallback: () => Promise<void>) => {
+const currentTagViewButtons = ({onPush, onSync, onEdit}: ButtonHandlers, getTagsCallback: () => Promise<void>) => {
 	const container = document.createElement("div");
 	container.id = TAG_LIST_BUTTON_CONTAINER_ID;
-	onLoggedIn(
-		async () => {
-			container.append(
-				createTagButton("Push", "img/book.png", onPush, "Add this author's tags to all of their books")
-			);
-			container.append(
-				createTagButton(
-					"Sync",
-					"img/enchanted-book.png",
-					onSync,
-					"Add this author's tags to all of their books, and delete extra tags"
-				)
-			);
-			container.append(createTagButton("Edit", "img/edit.gif", onEdit));
+	const onPushButton = createTagButton(
+		"Push",
+		"img/book.png",
+		onPush,
+		"Add this author's tags to all of their books"
+	);
+	const onSyncButton = createTagButton(
+		"Sync",
+		"img/enchanted-book.png",
+		onSync,
+		"Add this author's tags to all of their books, and delete extra tags"
+	);
+	const onEditButton = createTagButton("Edit", "img/edit.gif", onEdit);
+
+	onLogged({
+		container,
+		description: "Log in to manage this author's book tags",
+		onLogIn: async () => {
+			container.append(onPushButton, onSyncButton, onEditButton);
 			await getTagsCallback();
 		},
-		container,
-		"Log in to manage this author's book tags"
-	);
+		onLogOut: () => {
+			onPushButton.remove();
+			onSyncButton.remove();
+			onEditButton.remove();
+		},
+	});
```

Comment:
if only there was a multiple remove like the append above lol

Reply:
don't let ur dreams be dreams. pushed something but i'm passing out

will check when back from seattle.

or you can try and just merge it if it works